src/magic_folder/test/test_config.py

```diff
@@ -0,0 +1,166 @@
+from shutil import (
+    rmtree,
+)
+from os import (
+    mkdir,
+)
+
+from twisted.python.filepath import (
+    FilePath,
+)
+
+from testtools import (
+    ExpectedException,
+)
+from testtools.matchers import (
+    Equals,
+    NotEquals,
+)
+
+from .common import (
+    SyncTestCase,
+)
+from ..config import (
+    create_global_configuration,
+    load_global_configuration,
+)
+from ..snapshot import (
+    create_local_author,
+)
+
+
+class TestGlobalConfig(SyncTestCase):
+
+    def setUp(self):
+        super(TestGlobalConfig, self).setUp()
+        self.temp = FilePath(self.mktemp())
```

Comment:
A common thing to be broken is handling of non-ASCII paths.  It would be nice to exercise `create_global_configuration` with one of those.